Python/Product/PythonTools/PythonTools/Editor/PythonTextBufferInfo.cs

```diff
@@ -412,7 +411,7 @@ sealed class PythonTextBufferInfo {
         /// </summary>
         public IEnumerable<TrackingTokenInfo> GetTokens(ITextSnapshotLine line) {
             Tokenizer tokenizer = null;
-            var lineTokenization = GetLineTokenization(line, ref tokenizer);
+            var lineTokenization = _tokenCache.GetLineTokenization(line, () => tokenizer ?? (tokenizer = MakeTokenizer()));
```

Comment:
Should we just pass `Lazy<Tokenizer>` instead? And maybe change `MakeTokenizer` to return Lazy?

Reply:
Yep, makes sense